substanced/folder/generate/.gitignore

```diff
@@ -0,0 +1,2 @@
+/node_modules
```

Comment:
I don't have a strong opinion on the naming, just that it be outside the "normal" package tree.  Maybe `/jstooling` (peer of `substanced`)?  E.g.:

```
jstooling/
    generate/
setup.py
...
substanced/
    __init__.py
```